examples/source/interactivity-dynamic-content/Faux_Lightbox_Overlay.html

```diff
@@ -0,0 +1,96 @@
+<!---
+
+preview: amp4email
+author: fstanis
+
+--->
+
+<!--
+## Introduction
+
+This sample demonstrates how to create a simple lightbox-like effect using only [`amp-bind`](/documentation/components/amp-bind).
+
+It provides an alternative to using the [`amp-lightbox`](/documentation/components/amp-lightbox) component, in cases where it's inconvenient to use, for example emails with large height.
+-->
+
+<!-- -->
+<!doctype html>
+<html ⚡4email>
+<head>
+  <meta charset="utf-8">
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <script async custom-element="amp-bind" src="https://cdn.ampproject.org/v0/amp-bind-0.1.js"></script>
+  <style amp4email-boilerplate>body{visibility:hidden}</style>
+  <!--
+    ## Styles
+
+    Use CSS to define the styles for a background overlay and individual overlays displayed when a lightbox is open.
+  -->
+  <style amp-custom>
+    #lightbox-background {
+      position: absolute;
+      background-color: rgba(255, 255, 255, 0.9);
+      left: 0;
+      right: 0;
+      top: 0;
+      bottom: 0;
+    }
+
+    #lightbox-background.open {
+      display: block;
+    }
+
+    .lightbox-container {
+      position: relative;
+    }
+
+    .lightbox-overlay {
+      position: absolute;
+      top: 0;
+      z-index: 1;
+    }
+  </style>
+</head>
+<!-- ## Implementation -->
+<!-- -->
+<body>
+  <!--
+    Wrap the initially visible content as well as overlay content into an element with the `lightbox-container` class.
+
+    We use the `overlays` object to hold the state of all overlays on the page. Each overlay has a separate property inside this object it binds to, in this case `overlays.o1`.
+
+    Pressing the "Show overlay" button changes the state, displaying both the overlay and the background on the whole page.
+  -->
+  <div class="lightbox-container">
```

Comment:
Would it be possible to add the class to the body?

Reply:
No - the container is used as base for positioning, i.e. the location on which the overlay should be made on top of. Having it on the body would just result in the overlay being on top of the whole document (similar to lightbox).

Reply:
Hm, this doesn't really sound like a solution (or I didn't understand it correctly). Wouldn't this vary depending on desktop vs mobile viewports?

Reply:
The idea is to display a dismissable overlay on top of another element (i.e. use the same position as that element) while at the same time displaying a lightbox-like effect, hiding the rest of the document.